### ISSUE TITLE
O2KR2 - Feedback tool to enable people to give qualitative feedback to others

### ISSUE THREAD

**Issue by ceripower** [status-im/people-ops#8]

**Comment by jonathanbarker** [comment]
Merging https://github.com/status-im/people-ops/issues/21 into this issue. 

Solution for this issue could include a Kudos token or similar. Idea:  https://discuss.status.im/t/contributor-kudos-at-the-town-hall/803/5

**Comment by ceripower** [comment]
@rajanie @jonathanbarker @naghdy - been thinking about this OKR and what we might want to do next on it. Questions for you:

1. Do you think quantitative feedback (like peer review) is useful/valuable in our context?
2. Based on the above, is a feedback process/system worth implementing? [Could even be on an opt in basis, so people can use it if they want, but don't have to].

I took another look at what's available in BambooHR and the performance module is pretty lightweight and allows someone to write a quick self assessment, which it then pings on to their team lead for their review. There's also an option to request peer feedback from chosen team members. More [here](https://www.bamboohr.com/performance-management/) about how it works. 

There is a goals tracking functionality, but I think that's superfluous given that we track OKRs elsewhere. Good news is that that can be deactivated, leaving only the `assessments` plus `peer feedback` elements.

Before looking into this further I think it's useful to gauge whether this is something we want to pursue, or if this OKR should be dropped. Thank you!

**Comment by ceripower** [comment]
<img width="666" alt="screen shot 2019-02-14 at 08 25 09" src="https://user-images.githubusercontent.com/25318254/52773987-2d6c0f80-3034-11e9-8c5c-b9cb8ced2403.png">

**Comment by ceripower** [comment]
Example notification email:

<img width="720" alt="screen shot 2019-02-14 at 09 03 36" src="https://user-images.githubusercontent.com/25318254/52775376-770a2980-3037-11e9-84fd-727e5c7beb1b.png">

**Comment by jonathanbarker** [comment]
I think given we're launching Kudos Rewards (which will provide a form of quantitative feedback) soon, it's probably the wrong time to launch a qualitative feedback system as well.  I also think that the Bamboo style/language is to corporate/traditional to work for us. 

Maybe revisit for Q2 once we see how Kudos Rewards is going?

**Comment by ceripower** [comment]
> I think given we're launching Kudos Rewards (which will provide a form of quantitative feedback) soon, it's probably the wrong time to launch a qualitative feedback system as well

Yeah I hear you, the timing feels a bit off. Another thing to consider is that when we run the engagement survey, we can adapt what's on deck based on feedback that comes in. If lack of feedback mechanism comes up repeatedly, we can re-prioritise it.

**Comment by ceripower** [comment]
As we gear up for Q2s OKRs, does this feel any more relevant/timely for inclusion in the next quarter. Thoughts @rajanie and @jonathanbarker ?

Feedback rounds are underway and seem to be working ok, if a bit high touch. Meritocracy DApp is not as far along as we anticipated, so maybe we're not quite ready to look into this.

**Comment by rajanie** [comment]
Sorry I dont think I understand your question. Do you just want to do the feedback rounds (Design one) and not have a feedback tool?

**Comment by ceripower** [comment]
Sorry for the confusing question! Was mainly trying to add some context. I think feedback rounds can work well for the next quarter (Q2), and see after that if a tool is needed, wdyt?

**Comment by rajanie** [comment]
Yeah, sounds good to me

**Comment by jonathanbarker** [comment]
Sounds good - if the feedback rounds work well, perhaps the plan could be to conduct feedback rounds for teams that want/need them the most in Q2 (assuming we can't scale feedback to all teams).
